apache-httpclient4/src/test/java/com/baeldung/client/ClientLiveTest.java

```diff
@@ -36,7 +36,7 @@
 class ClientLiveTest extends GetRequestMockServer {
 
 
-    @Test
+    /*@Test
```

Comment:
Did we comment out these tests because they were not needed? If yes please remove.